org.adempiere.base/src/org/compiere/model/MPeriod.java

```diff
@@ -672,7 +672,19 @@ public boolean isOpen (String DocBaseType, Timestamp dateAcct)
 			return false;
 		}
 
-		MAcctSchema as = MClient.get(getCtx(), getAD_Client_ID()).getAcctSchema();
+		MAcctSchema as = null;
+		MAcctSchema[] asList = MAcctSchema.getSchemasByOrg(getCtx(), getAD_Client_ID(), getAD_Org_ID(), get_TrxName());
+		if(asList != null && asList.length > 0) {
+			for(MAcctSchema schema:asList) {
+				as = schema;
+				if(schema.isAutoPeriodControl())
+					break;
+			}
+		}
+		
+		if(as == null)
+			as = MClient.get(getCtx(), getAD_Client_ID()).getAcctSchema();
+
 		if (as != null && as.isAutoPeriodControl())
 		{
```

Comment:
hi @igorpojzl ,

The current model is actually very open ended. For e.g, with 2 org, you can have:
1. 2 acctschema with orgonly_id for each org.
2. 2 acctschema with a tenant schema for all and an org level schema with orgonly_id for second org.
3. 3 acctschema with 1 schema for org1 and 2 schema for org2.

I guess to be complete, you should check client level schema and all org level schema with orgonly_id = ad_org_id.